src/Lib/Effects/User.hs

```diff
@@ -38,7 +40,8 @@ data User = User
 instance ToJSON User
 instance FromJSON User
 
-instance FromField PasswordHash
+instance FromField PasswordHash where
+  fromField f mbs = PwdHash <$> fromField f mbs
```

Comment:
Maybe it's better to use `GeneralizedNewtypeDeriving` and move this instance to `PasswordHas` directly using `deriving`? And also remove `-fno-warn-orpanc` warning from this module.